metabase/metabase
# add new tooltip to the line/area/bar/combo charts

Part of https://github.com/metabase/metabase/pull/27394 extracted to simplify the review process

## Changes

Adds the new tooltip to line/area/bar/combo/waterfall/scatter charts.

## Screenshots

<img width="858" alt="Screen Shot 2023-01-05 at 2 43 54 PM" src="https://user-images.githubusercontent.com/14301985/210901684-77203d8b-9a25-4f65-a503-df4207f44f8a.png">
<img width="840" alt="Screen Shot 2023-01-05 at 2 44 04 PM" src="https://user-images.githubusercontent.com/14301985/210901701-6360abfe-a1a6-403f-afc8-df89026182e1.png">

## How to verify

- create line/area/bar/combo/waterfall/scatter charts and ensure the tooltip looks as expected
- create two line/area/bar/combo charts with the same dimensions and combine them on a dashboard
- ensure the tooltip shows correct labels and shows correct data for hovered series
